### PR TITLE
feat: add og subgraph queries

### PR DESCRIPTION
**Pull request by daywiss**
### Issues
we recently deployed an OG subgraph. We want to add queries to it to further optimize calls when requesting historical data.

Fixes #

### Changes 
*_(Briefly describe the changes made in this PR)_*

1. this removes all the event based queries and replaces them with calls to the OG subgraph
2. this also fixes an issue with incorrect data type being returned from subgraph query, number changed to bignumber
3. this also updates to production subgraph points, and updates any missing contract addresses in constants

### How to test
*_(Explain how the changes can be tested, including any required setup steps)_*

1. go to the proposal: #/barnbridge.eth/proposal/0x74cf6f3372aaeebfc2e99ff54604a5e63ca3d44cb84bdbf202a85581dfac9f5d
see that there is no warning in console Error querying module details from the graph. this means it used the subgraph function successfully. it shows that it has been executed.

### To-Do
*_(List any outstanding tasks be addressed before or after this PR is merged)_*

### Self-review checklist
- [x] I have performed a full self-review of my changes
- [x] I have tested my changes on a preview deployment
- [ ] I have tested my changes on different screen sizes (sm, md)
- [ ] I have tested my changes on a custom domain

### Additional notes or considerations
*_(Include any other relevant information or context that may be helpful for the reviewer)_*

### CODE REVIEW

File: src/plugins/safeSnap/utils/umaModule.ts [hunk 1]
```diff
@@ -274,26 +300,25 @@ export const getModuleDetailsUma = async (
...
+
+      return {
+        assertionId:event?.args?.assertionId,
+        expirationTimestamp: assertion.expirationTime,
+        isExpired: isExpired,
+        isSettled: assertion.settled,
+        proposalHash: proposalHash,
+        proposalTxHash: event.transactionHash,
+        logIndex: event.logIndex
+      };
```
Comment (daywiss) [inline src/plugins/safeSnap/utils/umaModule.ts:321]: this was just re-formatted to use async await

File: src/plugins/safeSnap/utils/umaModule.ts [hunk 2]
```diff
@@ -422,53 +447,11 @@ export const getModuleDetailsUmaGql = async (
...
-      maxRange,
-      ({ start, end }: { start: number; end: number }) => {
-        return moduleContract.queryFilter(
-          moduleContract.filters.ProposalExecuted(proposalHash),
-          start,
-          end
-        );
-      }
-    )
-  ]);
```
Comment (daywiss) [inline src/plugins/safeSnap/utils/umaModule.ts:467]: we can remove all events now from the graph function call now that we have OG subgraphs

**Comment by daywiss** [comment]
we are making an update to the subgraphs, which will require some changes to this pr

File: src/plugins/safeSnap/constants.ts [hunk 3]
```diff
@@ -448,18 +450,30 @@ export const contractData: ContractData[] = [
...
     subgraph:
-      'https://api.thegraph.com/subgraphs/name/md0x/polygon-optimistic-governor'
+      'https://api.thegraph.com/subgraphs/name/umaprotocol/polygon-optimistic-governor'
   },
   {
     network: '42161',
     name: 'OptimisticGovernor',
     subgraph:
-      'https://api.thegraph.com/subgraphs/name/md0x/arbitrum-optimistic-governor'
+      'https://api.thegraph.com/subgraphs/name/umaprotocol/arbitrum-optimistic-governor'
```
Comment (daywiss) [inline src/plugins/safeSnap/constants.ts:459]: adding production endpoints

File: src/plugins/safeSnap/components/HandleOutcomeUma.vue [hunk 4]
```diff
@@ -235,14 +235,15 @@ const questionState = computed<QuestionState>(() => {
...
+  if (!assertionEvent?.isExpired) return 'waiting-for-liveness';
 
   // Proposal is approved if it expires without a dispute and hasn't been settled.
-  if (assertionEvent.isExpired && !assertionEvent.isSettled)
+  if (assertionEvent?.isExpired && !assertionEvent?.isSettled)
     return 'proposal-approved';
 
   // Proposal is approved if it has been settled without a disputer and hasn't been executed.
-  if (assertionEvent.isSettled && !proposalExecuted) return 'proposal-approved';
+  if (assertionEvent?.isSettled && !proposalExecuted)
```
Comment (daywiss) [inline src/plugins/safeSnap/components/HandleOutcomeUma.vue:245]: ensuring this exists

**Review by bonustrack — APPROVED** [review]
tACK

**Pull request merged**